runtime/lua/vim/_meta/api_keysets_extra.lua

```diff
@@ -96,20 +96,29 @@ error('Cannot require a meta file')
 --- @field strikethrough? true
 --- @field altfont? true
 --- @field nocombine? true
-
---- @class vim.api.keyset.hl_info.cterm : vim.api.keyset.hl_info.base
 --- @field ctermfg? integer
 --- @field ctermbg? integer
+
+--- @class vim.api.keyset.hl_info.cterm : vim.api.keyset.hl_info.base
 --- @field foreground? integer
 --- @field background? integer
 
---- @class vim.api.keyset.hl_info : vim.api.keyset.hl_info.base
+--- @class vim.api.keyset.get_hl_info : vim.api.keyset.hl_info.base
 --- @field fg? integer
 --- @field bg? integer
 --- @field sp? integer
 --- @field default? true
+--- @field blend? integer
+--- @field cterm? vim.api.keyset.hl_info.cterm
+
+--- @class vim.api.keyset.set_hl_info : vim.api.keyset.hl_info.base
```

Comment:
🤔 @MariaSolOs where is this used? (why was it created?)
A [quick search](https://github.com/search?q=repo%3Aneovim%2Fneovim+set_hl_info&type=code) doesn't give any usage result 🤔

Reply:
Hmm good point, I don't remember exactly what happened in this PR but it seems like `vim.api.keyset.highlight` is a type that includes both `vim.api.keyset.get_hl_info` and `vim.api.keyset.set_hl_info`.